e2e/support/helpers/e2e-dashboard-helpers.js

```diff
@@ -1,5 +1,5 @@
 import { visitDashboard } from "./e2e-misc-helpers";
-import { menu, popover } from "./e2e-ui-elements-helpers";
+import { menu, popover, sidebar } from "./e2e-ui-elements-helpers";
```

Comment:
The value depends on the operator. If we're changing operator from multiple-value one to a single-value one, the default value still mentions 2 values

https://github.com/metabase/metabase/assets/6830683/a0aff875-d2ee-46c1-8eb1-14fb23b28114

Reply:
Similarly, when changing the filter type, the value can get "broken"

https://github.com/metabase/metabase/assets/6830683/c620e55e-f32d-44ea-a558-e06af02d1d43

Reply:
I think the simplest solution would be to clear the value if it cannot be used after changing filter type or the operator.

Reply:
Oh, now I see you covered changing the filter type in #41486.
But we still should cover changing the operator type.

Reply:
you're right, there should be some additional logic to understand if default value can be re-used. but I think the simplest solution for now is to drop default value in both cases